src/ibex_bluesky_core/callbacks/file_logger.py

```diff
@@ -38,17 +38,26 @@
 class HumanReadableFileCallback(CallbackBase):
     """Outputs bluesky runs to human-readable output files in the specified directory path."""
 
-    def __init__(self, fields: list[str], *, output_dir: Path = DEFAULT_PATH) -> None:
+    def __init__(
+        self, fields: list[str], *, output_dir: Path = DEFAULT_PATH, postfix: str = ""
```

Comment:
slightly pedantic, but the example in the docs for this callback doesn't specify a postfix, could you add one pls